docs/understanding-tracking-design/managing-event-specifications/ui/index.md

```diff
@@ -0,0 +1,87 @@
+
+Your event specification guarantees compatibility with the associated data structure version. This ensures events tracked using specified instructions pass validation for the associated data structure.
+
+If a newer event specification version is deployed, consider upgrading event specification and tracking to the latest data structure, ensuring ongoing compliance with validation requirements.
+
+## Creating and Editing Event Specifications
+
+To create a new event specification, follow these steps:
+
+1. Select a data product
+2. Click the "Create event" button
+3. A modal will appear, allowing you to enter the event name
+4. Your first event specification will be displayed on the page
+
+![Create an Event Specification](images/create-event-specification.png)
+
+To add more information or modify an existing event specification, follow these steps:
+
+1. Navigate to the appropriate data product
+2. Select the desired event specification
+3. This action will open a page displaying the details of the selected event specification
+
+This page is divided into different sections; below, you'll find more details for each section.
+
+![Example of an Event Specification Overview](images/event-specification-overvioew.png)
+
+### Event Information
+
+This section provides essential meta-information for your event specification, including the event name, description, and the applications in which this event is tracked.
+
+### Event Data Structure
+
+This section defines the structure of your data and specifies the types of properties associated with this event, along with its version.
+
+You can choose from two types of data structures:
+
+- **Standard**; Provided by the SnowPlow tracker by default
+- **Custom**; Provided by your organization
+
+![Event Data Structure](images/event-data-structure.png)
+
+## Entity Data Structures
+
+Within this section, you have the flexibility to choose the specific entities that should be associated with the event when it is triggered.
+
+Moreover, the section not only allows you to select entities but also provides comprehensive details about the properties associated with these entities and their respective versions.
+
+![Entity Data Structures](images/entity-data-structures.png)
+
+### Event Triggers
+
+This section defines the locations and circumstances under which this event is triggered.
+
+To create your first trigger, simply select the "Add trigger" button. To edit an existing trigger, click on the corresponding item in the trigger list.
+
+A modal will appear, allowing you to upload an image and provide additional context, such as the URL to the page on which this trigger applies.
+
+![Event Triggers](images/event-triggers.png)
+
+### Implementation Instructions
+
+The Implementation Instructions section allows you to specify how each property for a selected data structure should be populated.
+
+The dialog displays the list of properties defined in the data structure associated with this event specification.
```

Comment:
We seem to use both `modal` and `dialog` in different places. Would it make more sense to stick with `dialog` as maybe that is more commonly understood?

Reply:
done, from modal to dialog, included in this PR only changes in the data products section, in a separate PR I will replace all over modal text, thanks for the suggestion